week-1/Homework/mandatory/4-tax.js

```diff
@@ -5,7 +5,14 @@
   Sales tax is 20% of the price of the product
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(productPrice) {
+
```

Comment:
Well done. This works but can you think of anything else to make break down this code so it's more readable. Imagine your work colleague doesn't know what 0.20 represents. What can you do to make code readable.